lib/utils/ignore.ts

```diff
@@ -227,7 +227,12 @@ async function listFiles(projectDir: string): Promise<string[]> {
 					const _files = await fs.readdir(dir, { withFileTypes: true });
 					for (const entry of _files) {
 						const fpath = path.join(dir, entry.name);
-						if (entry.isDirectory()) {
+						const isDirectory =
+							entry.isDirectory() ||
+							(entry.isSymbolicLink() &&
+								(await fs.stat(await fs.realpath(fpath))).isDirectory());
```

Comment:
Is the `realpath()` call necessary? I understand that `fs.stat()` will (recursively) follow symlinks if `fpath` is a symlink (as opposed to `fs.lstat()`), so it would not be necessary to explicitly resolve the symlink with `realpath()`.

```suggestion
								(await fs.stat(fpath)).isDirectory());
```

Reply:
TIL that `fs.stat()` does follow symbolic links. Nice catch :+1: